task/validate-fbc/0.1/validate-fbc.yaml

```diff
@@ -216,9 +216,9 @@ spec:
             - SETFCAP
       computeResources:
         limits:
-          memory: 4Gi
+          memory: 6Gi
```

Comment:
Could we please set the ratio of memory request and limit to 1:1?

Setting it different can be detrimental to other tasks.

Reply:
Sure, I can update that. We didn't have that ratio before.

Reply:
yeah, ideally we should have that ratio set to 1:1 in all tasks. well, one battle at a time